packages/wonder-blocks-search-field/src/components/search-field.stories.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import * as React from "react";
+
+import type {StoryComponentType} from "@storybook/react";
+
+import SearchField from "../index.js";
+
+export default {
+    title: "SearchField",
```

Comment:
If you include `component: SearchField` here, SB Docs can use that to include the Props table in the `Docs` tab.

For reference: https://5e1bf4b385e3fb0020b7073c-hxnnfisaue.chromatic.com/?path=/docs/cell-compactcell--default-compact-cell

NOTE: You might need to add another key...`argTypes` that is used to customize how each prop is rendered in the Props table (or might not).